src/electron/release_linux_action.sh

```diff
@@ -22,10 +22,14 @@ yarn do src/electron/package_common
 
 scripts/environment_json.sh -r -p linux > www/environment.json
 
+# Set-up electron to only download new updates but never upload new ones
```

Comment:
Maybe this is clearer: "Build client binary. It configures the auto update location, but doesn't publish the binary."